docs/contents/components/forms/native-select/accessibility.ja.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: NativeSelect
+description: >-
+  `NativeSelect`は、ユーザーがオプションのリストから1つを選択するために使用されるコンポーネントです。ブラウザ（ユーザーエージェント）によって提供されるネイティブなドロップダウンリストを表示します。
+package_name: "@yamada-ui/native-select"
+order: 4
+with_description: true
+tab: アクセシビリティ
+---
+
+`NativeSelect` は、アクセシビリティに関して、[WAI-ARIA - Combobox Pattern]("https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-select-only/")に従います。
+
+## キーボード操作
+
+| キー             | 説明                                                                                                       | 状態 |
+| ---------------- | ---------------------------------------------------------------------------------------------------------- | ---- |
+| `Tab`            | フォーカスがセレクトボックスに移動すると、最初の項目をフォーカスします。                                   | -    |
+| `ArrowUp`        | 無効ではない前のオプションをフォーカスします。最初の項目の場合は無効ではない最後の項目をフォーカスします。 | -    |
+| `ArrowDown`      | 無効ではない次のオプションをフォーカスします。最後の項目の場合は無効ではない最初の項目をフォーカスします。 | -    |
+| `Space`, `Enter` | フォーカスしているオプションをアクティブにします。                                                         | -    |
+| `Home`           | 無効ではない最初のオプションをフォーカスしてアクティブにします。                                           | -    |
+| `End`            | 無効ではない最後のオプションをフォーカスしてアクティブにします。                                           |
+
+## ARIAロールと属性
+
+| コンポーネント           | ロールと属性         | 使い方                                                   |
+| ------------------------ | -------------------- | -------------------------------------------------------- |
+| `NativeSelect`           | `aria-hidden="true"` | 要素をアクセシビリティツリーから除外します。             |
+| `select` <Tag>内部</Tag> | `aria-readonly`      | `isReadOnly`が設定されている場合は`"true"`を設定します。 |
```

Comment:
- Please add role.
- Where is `aria-hidden`?

![image](https://github.com/user-attachments/assets/53a614b4-b8b2-4321-99bc-6dd446ba1d31)

Reply:
I assumed aria-hidden=true would work based on the following code. Is this incorrect?" https://github.com/yamada-ui/yamada-ui/blob/46c97e371adc766101b2a1ef5d0c83270889f3ad/packages/components/native-select/src/native-select.tsx#L241

Reply:
This is `NativeSelectIcon`.
https://github.com/yamada-ui/yamada-ui/blob/46c97e371adc766101b2a1ef5d0c83270889f3ad/packages/components/native-select/src/native-select.tsx#L214-L251